misc-utils/uuidd.service.in

```diff
@@ -16,7 +16,7 @@ ProtectKernelTunables=yes
 ProtectKernelModules=yes
 ProtectControlGroups=yes
 MemoryDenyWriteExecute=yes
-ReadWritePaths=/var/lib/libuuid/
+StateDirectory=libuuid
```

Comment:
I wouldn't say I like this change. It makes things fragile as the rest of the code assumes "/var/lib/libuuid". 

We need to change the path in the service file to @localstatedir@/libuuid. 

Note that the path is hardcoded in libuuid. Using the same path everywhere would be more robust than only in service file follow any 3rd-party shorcut.  Imagine that I'll compile libuuid/uuidd with --localstatedir=/foo/bar than the file will be useless.

Reply:
I'll fix the hardcoded path in the service file (we need @localstatedir@/libuuid there).

Reply:
Honestly, I feel this case quite synthetic. If people don't follow normal prefix, then they are unlikely to use the service either... But OK.

Reply:
This is the difference between upstream and downstream. It's fine to assume anything in your distro and prepare a package for the distro, but for upstream it's difficult to assume anything and at the same time support a large range different usecases (and paths) and configurations.